aries-site/src/pages/components/controls.js

```diff
@@ -62,6 +67,9 @@ const Controls = () => (
     </ContentSection>
     <ContentSection>
       <Subsection name="Button">
+      <SubsectionText>
```

Comment:
This comment applies to all the examples on this page, but I think there had been discussion a month or so ago (before you joined on the project) where we decided the content order would go:
1. Section title
2. Example
3. SubsectionText

I realize that isn't updated in the designs but came out of discussion, and I forgot about that when we talked this morning. For all of these, can you keep the order as listed above?

Reply:
@taysea This order has had additional discussion / rationale. Offline discussions with Greg, plus finding the Slack convo for reference. It starts here + next 5-6 comments: https://hpe.slack.com/archives/GPPLUK6LR/p1581632692012300

Reply:
Gotcha! Thanks for pointing me to that. I missed that conversation. PR looks good to me then.